app/router.jsx

```diff
@@ -85,7 +70,7 @@ class ONE_UP_REDIRECT extends React.Component {
 function redirectAboutPage (nextState, replace, done) {
   try {
     const { pathname } = nextState.location
-    const newUrl = `https://fe-content-pages.zooniverse.org${pathname}`
+    const newUrl = `https://fe-root.preview.zooniverse.org${pathname}`
```

Comment:
I'm not sure this `newUrl` init is relevant anymore because ALL About pages and Get Involved pages are using FEM after this PR is deployed, but I kept it just in case 🤷‍♀️

Reply:
I think you're right, it won't be needed, but can't hurt to wait and remove after app-root deploy.